winpr/libwinpr/input/scancode.c

```diff
@@ -104,7 +129,426 @@ static DWORD KBD7X[128] = {
+	                              VK_LMENU,
+	                              VK_SPACE,
+	                              VK_CAPITAL,
+	                              VK_F1,
+	                              VK_F2,
+	                              VK_F3,
+	                              VK_F4,
+	                              VK_F5,
+	                              VK_F6,
+	                              VK_F7,
+	                              VK_F8,
+	                              VK_F9,
+	                              VK_F10,
+	                              VK_NUMLOCK,
+	                              VK_SCROLL,
+	                              VK_HOME,
+	                              VK_UP,
+	                              VK_PRIOR,
+	                              VK_SUBTRACT,
+	                              VK_LEFT,
+	                              VK_CLEAR,
+	                              VK_RIGHT,
+	                              VK_ADD,
+	                              VK_END,
+	                              VK_DOWN,
+	                              VK_NEXT,
+	                              VK_INSERT,
+	                              VK_DELETE,
+	                              VK_SNAPSHOT,
+	                              VK_OEM_102,
+	                              VK_F11,
+	                              VK_F12,
+	                              VK_CLEAR,
+	                              VK_OEM_WSCTRL,
+	                              VK_DBE_KATAKANA,
+	                              VK_OEM_JUMP,
+	                              VK_DBE_FLUSHSTRING,
+	                              VK_OEM_BACKTAB,
+	                              VK_OEM_AUTO,
+	                              VK_NONE,
+	                              VK_DBE_NOCODEINPUT,
+	                              VK_HELP,
+	                              VK_NONE,
+	                              VK_F13,
+	                              VK_F14,
+	                              VK_F15,
+	                              VK_F16,
+	                              VK_F17,
+	                              VK_F18,
+	                              VK_F19,
+	                              VK_F20,
+	                              VK_F21,
+	                              VK_F22,
+	                              VK_F23,
+	                              VK_OEM_PA3,
+	                              VK_NONE,
+	                              VK_OEM_RESET,
+	                              VK_NONE,
+	                              VK_ABNT_C1,
+	                              VK_NONE,
+	                              VK_NONE,
+	                              VK_F24,
+	                              VK_NONE,
+	                              VK_NONE,
+	                              VK_NONE,
+	                              VK_NONE,
+	                              VK_OEM_PA1,
+	                              VK_TAB,
+	                              VK_ABNT_C2,
+	                              VK_OEM_PA2 };
+
+static const DWORD KBD8X[256] = { VK_NONE,
```

Comment:
This (`KBD8X`) and `KBD8T` should better use the same formatting, as it was done in the other already existing arrays (`KBD4X`, `KBD4T`, `KBD7X`, `KBD7T`)

Reply:
oh, looks like a `clang-format` issue, need to check.

Reply:
indeed.
did disable the automatic formatting for these two arrays as the formatting depends on the length of the define names (tires to neatly arrange these in rows, but fails doing that)